docs/concepts/backup-restore.md

```diff
@@ -1,34 +1,38 @@
-# Backup and restore
+---
+title: Backup and Restore
+---
```

Comment:
Additionally, if it is required currently `docs/concepts/admission-controller.md`, `docs/concepts/architecture.md`, `docs/concepts/controller-manager.md`, `docs/concepts/network-policies.md`, `docs/concepts/operator.md`, `docs/concepts/resource-manager.md` and `docs/concepts/scheduler.md` do not specify such section.

Reply:
The current way the title of gardener/gardener topics gets generated is via capitalization of the document name. In certain cases, this creates a name that does not reflect the actual document title (Backup Restore, Etcd, Apiserver, ...). Adding a title in the frontmatter of the document is how we fix this. It would be possible to add the title as frontmatter to all documents, regardless of document name.

The contribution guide has a section on [frontmatter](https://gardener.cloud/docs/contribute/20_documentation/40_style_guide/#front-matter).